### PR TITLE
Add password recovery in FAQ

### PR DESCRIPTION
**Pull request by lemelon**

### CODE REVIEW

**Comment by frankrousseau** [comment]
@clochix could you review texts from this PR?

**Comment by frankrousseau** [comment]
@clochix Can you review this PR? That way, we would be able to add it to the new version?

File: server/locales/fr.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   "error login failed": "Echec de la connexion.",
   "reset password email from": "Votre Cozy <no-reply@%{domain}>",
   "reset password email subject": "[Cozy] Réinitialiser votre mot de passe",
-  "reset password email text": "Il semble que vous ayez oublié le mot de passe de votre Cozy.\nNe vous inquiétez pas, suivez simplement le lien suivant pour en définir un nouveau :\n\nhttps://%{domain}/password/reset/%{key}\n\nN'oubliez pas de mettre à jour toutes vos données chiffrées après ça !",
+  "reset password email text": "Il semble que vous ayez oublié le mot de passe de votre Cozy.\nNe vous inquiétez pas, suivez simplement le lien suivant pour en définir un nouveau :\n\nhttps://%{domain}/password/reset/%{key}\n\nN'oubliez pas de mettre à jour toutes vos données chiffrées après ça !\n\nFor more info, please visit the FAQ of cozy :\n\nhttps://cozy.io/fr/faq.html#que-faire-si-j-oublie-mon-mot-de-passe-\n\n",
```
Comment (clochix) [inline server/locales/fr.json]: @lemelon could you use french here ?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cozy-proxy",
-  "version": "2.1.3",
+  "version": "2.1.4",
```
Comment (clochix) [inline package.json]: Why are you upgrading the application version ? Usually we only do this before publishing the application.

**Pull request merged**